libraries/core_libs/consensus/include/transaction_manager/transaction_manager.hpp

```diff
@@ -52,6 +52,14 @@ class TransactionManager : public std::enable_shared_from_this<TransactionManage
    */
   uint32_t insertBroadcastedTransactions(const SharedTransactions &txs);
 
+  /**
+   * @brief Checks the cache if transaction is already seen
+   *
+   * @param trx_hash transaction hash
+   * @return true if seen
+   */
+  bool transactionSeen(const trx_hash_t &trx_hash);
```

Comment:
bool transactionSeen(const trx_hash_t &trx_hash) **const**;

Reply:
Fixed

Reply:
how is this fixed ? You are still missing const at the end of function declaration